NOTES-NONSTOP.md

```diff
@@ -78,12 +78,64 @@ The current OpenSSL default memory model uses the default platform address
 model. If you need a different address model, you must specify the appropriate
 c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
 
+Cross Compiling on Windows
+--------------------------
+
+To configure and compile OpenSSL, you will need to set up a Cygwin environment.
+The Cygwin tools should include bash, make, and any other normal tools required
+for building programs.
+
+The `PATH` must include the bin directory for the c99 cross-compiler, as in:
```

Comment:
Should it be _Your_ instead of _The_?  The rest is using the second person, so it seems natural to do so here.